modules/ROOT/pages/_partials/rn-deprecated-ref.adoc

```diff
@@ -1,5 +1,4 @@
 [id='rn-deprecated-ref']
 = Deprecated features
 
-// This is https://issues.redhat.com/browse/INTLY-5350
-* Support for TLS 1.0 and TLS 1.1 is removed from {PRODUCT}. Only TLS 1.2 is supported as of this release.
+There are no deprecated features in this release.
```

Comment:
If there is nothing, should we have this section at all?

Reply:
Susan and I asked the same question. We decided to keep the sections that have no updates for the sake of consistency with older release notes.